packages/compiler/src/render3/view/template.ts

```diff
@@ -914,7 +915,7 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
     // Only add normal input/output binding instructions on explicit <ng-template> elements.
     if (template.tagName === NG_TEMPLATE_TAG_NAME) {
       const [i18nInputs, inputs] = partitionArray(template.inputs, hasI18nMeta);
```

Comment:
FYI, tests in Google's codebase returned the following build failure:
```
packages/compiler/src/render3/view/template.ts:931:54 - error TS2345: Argument of type '(Node & { i18n?: Message | Node | undefined; })[]' is not assignable to parameter of type '(TextAttribute | BoundAttribute)[]'.
  Type 'Node & { i18n?: Message | Node | undefined; }' is not assignable to type 'TextAttribute | BoundAttribute'.
    Type 'Node & { i18n?: Message | Node | undefined; }' is not assignable to type 'BoundAttribute'.

931         this.templatePropertyBindings(templateIndex, inputs);
                                                         ~~~~~~
```
changing the code as follows resolved the issue:
```suggestion
      const [i18nInputs, inputs] = partitionArray<t.BoundAttribute, t.BoundAttribute>(template.inputs, hasI18nMeta);
```
@crisbeto could you please apply this update? Thank you.

Reply:
Done.

Reply:
Thanks @crisbeto 👍